scripts/globals/besieged.lua

```diff
@@ -216,15 +216,46 @@ xi.besieged.getAstralCandescence = function()
     return 1 -- Hardcoded to 1 for now
 end
 
-xi.besieged.badges = { 780, 783, 784, 794, 795, 825, 826, 827, 894, 900, 909 }
+xi.besieged.badges =
```

Comment:
Just curious, is there a reason we're keeping this in besieged still (and the supporting functions) now that assault has a separate global?

Reply:
No clue, but since it's already in here I didnt want to start a full rewrite